keyboards/cannonkeys/satisfaction75_hs/satisfaction75_hs.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 Andrew Kannan
```

Comment:
This file can be deleted.

Reply:
This is necessary in order to not have to import the keycodes in each keymap file. (See drashna's feedback above)